### PR TITLE
Updated the first timers thanks contribution bot

### PR DESCRIPTION
**Pull request by Josephchinedu**
Signed-off-by: Josephchinedu <47852925+Josephchinedu@users.noreply.github.com>

**Description**
i added " &nbsp;
 &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9;   :star: Please [star the project](../stargazers) if you have yet to do so."
to .github/config.yml:

This PR fixes #

**Notes for Reviewers**
I updated the first timers thanks contribution bot

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -19,7 +19,8 @@ firstPRMergeComment: >
    
   ![Congrats!](https://raw.githubusercontent.com/layer5io/layer5/master/.github/welcome/Layer5-celebration.png)
 
-  Please [star the project](../stargazers) if you have yet to do so.
+   &nbsp;
```
Comment (PluckyPrecious) [inline .github/config.yml:22]: Hi @Josephchinedu I don’t think there should be any space in line 22 [see this](https://github.com/layer5io/service-mesh-performance/blob/master/.github/config.yml)

**Comment by PluckyPrecious** [comment]
Weldone @Josephchinedu kindly make the requested change,  edit the description & mention issue fixed - This PR fixes #issuenumber, also the second commit is it part of this issue or another? I think another contributor made a PR on that already.

**Review by Jashpatel1 — CHANGES_REQUESTED** [review]
Looks great

**Review by PluckyPrecious — COMMENTED** [review]
> You may have to remove the calendar open change as a PR has already been made for that change. Keeping it on will lead to merge conflict.

The issue was assigned to him, but the other contributor just went ahead and worked on it without making it known.

**Review by leecalcote — APPROVED** [review]
thank you, @Josephchinedu. 👍

**Comment by leecalcote** [comment]
Thank you for PR #718, @Josephchinedu. 👍

**Pull request closed without merging**